server/region_handler_test.go

```diff
@@ -169,6 +169,15 @@ func (ts *TidbRegionHandlerTestSuite) TestGetRegionByIDWithError(c *C) {
 	defer resp.Body.Close()
 }
 
+func (ts *TidbRegionHandlerTestSuite) TestRegionsFromMeta(c *C) {
+	ts.startServer(c)
+	defer ts.stopServer(c)
+	resp, err := http.Get("http://127.0.0.1:10090/regions/meta")
+	c.Assert(err, IsNil)
+	defer resp.Body.Close()
+	c.Assert(resp.StatusCode, Equals, http.StatusOK)
```

Comment:
Why not verify the body?

Reply:
Now we do verify whether the body is a valid json array or not.

Reply:
Why not verify if it is region meta?

Reply:
Fixed. PTAL, thanks!